jargons/webpack.md

```diff
@@ -4,4 +4,14 @@ tags:
 - javascript
 ---
 
-[Node.js](/nodejs) ile yazılan kodun tarayıcıda da çalışabilmesi için gerekli olan bir araçtır.
+Webpack bir modül paketleyicisidir. Sizin kodunuzu ve o koddakı bağımlılıkları **bundle.js** adında tek bir Javascript dosyasına yazar/paketler (Aslında sadece Javascript bağımlılıklarınızı değil; css, fonts, images gibi bütün statik assetlerinizi paketleyebilir).
```

Comment:
fonts, images yerine türkçe karşılığı fontlar, resimler yazılması daha doğru olur sanki.

Reply:
`asset` kelimesine de jargon eklenip burada linklenebilir.

Reply:
webpack'in bağımlılıkları paketleme işi dışında yaptığı, küçültme, optimize etme vb işlevlerinden de bahsedilebilir.

Reply:
- fonts, images kelimelerini duzelttim.
- asset icin bir jargon eklendiginde linklemeyi yapabiliriz, bu branch'i mergelemek icin bir bloklayici oldugunu dusunmuyorum. (ya da siz jargonu ekleyip burayi guncelleyebilirsiniz?)

- aslinda webpack'in yaptigi cok is var, zamani olan biri ekleyebilir. benim asil amacim ilk hali cok yanlis olan tanimi degistirmekti, bu yuzden temel yaptigi seyi yazdim.